README.md

```diff
@@ -39,38 +38,97 @@ print(favoriteSentence)
 ```swift
 let name = "Ned Stark"  // the student has seen this
 ```
-* The student has NOT seen type annotations yet. It isn't something I want to have the student see (yet).
+
+## Printing Values
+
+When learning Swift, one of the most basic things you'll no doubt want to do is print stuff to the screen. Even once you know Swift inside and out, displaying strings, numbers, and other data on the screen is one of the most fundamental things you'll need to do on a regular basis, because what good is a program that never presents any information to its users?
+
+In Swift, you can print a variable or a constant to the screen using the `print()` function. Let's see that in action by creating a playground file, as you learned in the last lesson.
+
+After creating your new playground in Xcode, start by declaring three strings representing your favorite word, color, and musician. First ask yourself: Should these be _variables_ (declaring using `var`) or _constants_ (declared using `let`)? You probably feel pretty strongly about your favorite word, color, and musician, but your tastes may change over time, so let's declare them as variables for now. You should enter them in your playground file:
 
 ```swift
-let name: String = "Ned Stark"  // the student has not seen this
+var favoriteWord = "mercurial"    // Use your own favorite word!
+var favoriteColor = "blue"        // Use your own favorite color!
+var favoriteMusician = "Nirvana"  // Use your own favorite musician!
 ```
 
+You'll see your values represented in the sidebar on the right side of your playground:
+
+![Playground file](.images/xcode1.png)
 
+Why do you see output if you haven't even called the `print()` function yet? It's important to note that the right sidebar in a playground file doesn't represent program _output_; rather, it's a look at how your program is working and what values different variables and constants have. If this were a real program being run outside of a playground file, you wouldn't see any output at all. Playgrounds are special in that they show you the state of your program at given moments, even if you're not explicitly outputting any data.
 
```

Comment:
Could not have worded this any better. The student will definitely be questioning what that is in the right pane and this should help.